Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -2993,8 +2993,8 @@ IndexedDBAPIEnabled:
 InertAttributeEnabled:
   type: bool
   status: stable
-  category: css
-  humanReadableName: "inert attribute"
+  category: html
+  humanReadableName: "HTML inert attribute"
```

Comment:
Do we need to prefix the name with HTML if it's in the new HTML category?

Reply:
And if so, why don't you change the other labels, too?

Reply:
Wanted to make it consistent with the popover attribute. Regarding whether to put the prefix or not, the same question applies to the CSS labels as well.

Personally I like the prefix, because it keeps the label capitalized without having to capitalize a technical term.